CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # OverReact Changelog
 
+## 2.4.0
+
+> Complete `2.4.0` Changsets:
+>
+> - [Dart 2](https://github.com/Workiva/over_react/compare/2.3.1+dart2...2.4.0+dart2)
+> - Dart 1 (no changes)
+
+* [#280] Optimize the over_react builder to avoid unnecessary asset reads.
+  Informal profiling in a large codebase that consumes over_react has shown in
+  the best case a speed up on rebuilds from ~1 minute to ~2 seconds.
```

Comment:
#nit Might be worth noting that no trials showed a decrease in performance.

Reply:
Addressed in c5c2856